src/helpers/_focusable.scss

```diff
@@ -32,3 +32,39 @@
     background-color: $govuk-focus-colour;
   }
 }
+
+/// Focusable with box-shadow
+///
+/// Removes the visible outline and replace with box-shadow and background colour.
+/// Used for interactive text-based elements.
+///
+/// @access public
+
+@mixin govuk-focusable-text {
+  // When colours are overridden, for example when users have a dark mode,
+  // backgrounds and box-shadows disappear, so we need to ensure there's a
+  // transparent outline which will be set to a visible colour.
+
+  // Since Internet Explorer 8 does not support box-shadow, we want to force the user-agent outlines
+  @include govuk-not-ie8 {
+    outline: $govuk-focus-width solid transparent;
+    outline-offset: 0;
+  }
+  color: $govuk-text-colour;
+  background-color: $govuk-focus-colour;
+  // sass-lint:disable indentation
+  box-shadow: -5px -1px 0 1px govuk-colour("yellow"),
+              5px -1px 0 1px govuk-colour("yellow"),
+              -3px 1px 0 3px govuk-colour("black"),
+              3px 1px 0 3px govuk-colour("black");
+  // sass-lint:enable indentation
+  // When link is focussed, hide the default underline since the
+  // box shadow adds the "underline"
+  text-decoration: none;
+}
+
+@mixin govuk-focusable-text-link {
```

Comment:
This mixin seems to be performing a task very similar to `@mixin govuk-focusable-fill`. Do we need both? It would be good to have a comment to indicate what the difference between the two is.

Reply:
As discussed on Slack we're going to wait until we have everything merged before deciding how the mixins should look.